packages/in_app_purchase/in_app_purchase_android/android/src/main/java/io/flutter/plugins/inapppurchase/MethodCallHandlerImpl.java

```diff
@@ -374,6 +378,41 @@ private void updateCachedSkus(@Nullable List<SkuDetails> skuDetailsList) {
     }
   }
 
+  private void launchPriceChangeConfirmationFlow(String sku, MethodChannel.Result result) {
+    if (activity == null) {
+      result.error(
+          "ACTIVITY_UNAVAILABLE",
+          "launchPriceChangeConfirmationFlow is not available. "
+              + "This method must be run with the app in foreground.",
+          null);
+      return;
+    }
```

Comment:
This code block should be tested. The same goes for the `if (billingClientError(result))`, the assert and `if (skuDetails == null)` checks.